docs/basic-features/eslint.md

```diff
@@ -138,6 +138,14 @@ Similarly, the `--dir` flag can be used for `next lint`:
 next lint --dir pages --dir utils
 ```
 
+## Caching
+
+To improve performance, information of files processed by ESLint are cached by default. This is stored in the `.eslintcache` file in the root of your directory. If you include any ESLint rules that depend on more than the contents of a single source file and need to disable the cache, use the `--no-cache` flag with `next lint`.
```

Comment:
if `--cache` is enabled by default, should `.eslintcache` be added to `.gitignore` in the `create-next-app` templates?

Reply:
Yep that's a great point. However, I'm updating currently this currently to write to `.next/cache` instead which is already in the `.gitignore` :)